node/resolvers/Mutations/Users.ts

```diff
@@ -180,6 +180,14 @@ export const addUser = async (_: any, params: any, ctx: Context) => {
   } = ctx
 
   try {
+    const costCenter = await ctx.clients.organizations.getCostCenterById(
+      params.costId
+    )
+
+    if (!costCenter?.data?.getCostCenterById.name) {
```

Comment:
I was thinking, and the `getCostCenterById` can also return a `organization` field which is the organization ID that cost center belongs to.
We should actually make sure the organization ID associated with the cost center should be the same as the `orgId` provided in the request.
If the cost center is not part of the org, we should not be able to add a user to that cost center.